### DISCUSSION TITLE
Add support for new message notification sound

### DISCUSSION THREAD

**Discussion by mscofield0** [d99kris/nchat#306] [Ideas]
Would be nice to hear when we receive a message.

**Comment by semedilino-073** [comment]
That would be so nice! Completely agree with you

**Comment by Docbroke** [comment]
This can be achieved with something like this in ui.conf
```
desktop_notify_command=nchat_notify 'nchat-whatsapp' '%1: %2'
```
with file nchat_notify in your path
```
#!/bin/sh
notify-send "$@"
mpv <path to your choosen tone> 2> /dev/null
```

**Comment by d99kris** [comment]
I think @Docbroke has suggested a good way to achieve this given current nchat functionality. I will leave this feature request open though, and gauge the interest to maybe see if we should add this extra functionality.
